content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -157,6 +157,18 @@ pointing to the Pod's IP address. Also, Pod needs to become ready in order to ha
 record unless `publishNotReadyAddresses=True` is set on the Service.
 {{< /note >}}
 
+### Pod's setHostnameAsFQDN field {#pod-sethostnameasfqdn-field}
+
+When a Pod is configured to have fully qualified domain name (FQDN), its hostname is the short hostname. For example, if you have a Pod with the fully qualified domain name `busybox-1.default-subdomain.my-namespace.svc.cluster-domain.example`, then by default the `hostname` command inside that Pod returns `busybox-1` and  the `hostname --fqdn` command returns the full FQDN.
+
```

Comment:
nit: Does it make sense to remove `full` and just say,
... command returns the FQDN.

Reply:
i removed `full`. thanks